CONTRIBUTING.md

```diff
@@ -338,6 +338,14 @@ DOCKER_HUB_REPO=vmware EXTRA_TAG= VERSION_TAG=<version_tag> make all
 
 Update documentation following steps listed below.
 
+
+## Cutting release steps for vFile plugin
+### Publish vFile managed plugin to Docker Store
+```
+DOCKER_HUB_REPO=vmware EXTRA_TAG= VERSION_TAG=latest make build-vfile
```

Comment:
can you help me to understand here? 

1. I have looked the target and it seems it is not pushing to docker hub. can you please confirm whether following command pushes the image to docker hub or not?

`DOCKER_HUB_REPO=cnastorage EXTRA_TAG= VERSION_TAG=latest make build-vfile`

Reply:
so this PR should go in after https://github.com/vmware/docker-volume-vsphere/pull/1801/files

Reply:
Yeah now it should be able to push after 1801 is in.